lib/src/connector/connector.cc

```diff
@@ -317,7 +326,13 @@ void Connector::stopMonitoring()
     checkConnectionInitialization();
 
     if (!is_monitoring_) {
-        LOG_WARNING("The Monitoring Thread is not running");
+        if (!monitor_exception_) {
+            LOG_WARNING("The Monitoring Thread is not running");
+        } else {
+            LOG_DEBUG("The previous instance the Monitoring Thread caught and "
```

Comment:
`The previous instance the Monitoring Thread` doesn't read very clearly. Not quite sure what you meant there.